docs/updating/migration-to-cdn/react.md

```diff
@@ -0,0 +1,461 @@
+		<CKEditor
+			editor={ ClassicEditor }
+			config={ {
+				toolbar: {
+					items: [ 'undo', 'redo', '|', 'bold', 'italic' ],
+				},
+				plugins: [
+					Bold, Essentials, Italic, Mention, Paragraph, SlashCommand, Undo
+				],
+				licenseKey: '<YOUR_LICENSE_KEY>', // Or 'GPL'.
+				mention: {
+					// Mention configuration
+				},
+				initialData: '<p>Hello from CKEditor 5 in React!</p>',
+			} }
+		/>
+	);
+}
+
+export default App;
+```
+
+**After:**
+
+```jsx
+import { CKEditor, useCKEditorCloud } from '@ckeditor/ckeditor5-react';
+
+function App() {
+	// Load CKEditor 5 from the CKEditor Cloud, using the `useCKEditorCloud` hook.
+	// It'll inject the CKEditor 5 scripts and styles into your document head and after
+	// successful loading, it'll return the `CKEditor` and `CKEditorPremiumFeatures` objects.
+	const cloud = useCKEditorCloud( {
+		version: '{@var ckeditor5-version}', // Required. The version of CKEditor 5 to load.
+		premium: true // Optional. Set to `true` if you want to use premium features.
+	} );
+
+	if ( cloud.status === 'loading' ) {
+		return <div>Loading...</div>;
+	}
+
+	if ( cloud.status === 'error' ) {
+		console.error( cloud.error );
+
+		return <div>Error!</div>;
+	}
+
+	// Pick the CKEditor 5 plugins you want to use.
+	const {
+		ClassicEditor, Bold, Essentials, Italic,
+		Mention, Paragraph, SlashCommand, Undo
+	} = cloud.CKEditor;
+
+	const { SlashCommand } = cloud.CKEditorPremiumFeatures;
+
+	return (
+		<CKEditor
+			editor={ ClassicEditor }
+			config={ {
+				toolbar: {
+					items: [ 'undo', 'redo', '|', 'bold', 'italic' ],
+				},
+				plugins: [
+					Bold,
+					Essentials,
+					Italic,
+					Mention,
+					Paragraph,
+					SlashCommand,
+					Undo
+				],
+				licenseKey: '<YOUR_LICENSE_KEY>', // Or 'GPL'.
+				initialData: '<p>Hello from CKEditor 5 in React!</p>',
+			} }
+		/>
+	);
+}
+```
+
+#### Step 3 (Optional): Migrate the CKEditor&nbsp;5 React testing suite
+
+If you have any tests that use CKEditor&nbsp;5 objects, you need to update them to use the  `loadCKEditorCloud` function. Here is an example of migrating a test that uses the `ClassicEditor` object:
+
+**Before:**
+
+```javascript
+import { ClassicEditor, ... } from 'ckeditor5';
+import { AIAdapter, ... } from 'ckeditor5-premium-features';
+
+it( 'ClassicEditor test', () => {
+	render(
+		<CKEditor
+			editor={ ClassicEditor }
+			// ... other props
+		/>
+	);
+} );
+```
+
+**After:**
+
+```javascript
+// It may be counterintuitive that in tests you need to use `loadCKEditorCloud` instead of `useCKEditorCloud`.
+// The reason for this is that `useCKEditorCloud` is a React hook and can only be used in React components,
+// while tests are typically written as functions in testing suites. Therefore, in tests, you should use
+// the `loadCKEditorCloud` function to load CKEditor 5 from the CKEditor Cloud and obtain the necessary
+// CKEditor 5 objects. This allows you to properly test your CKEditor 5 integration without any issues.
```

Comment:
Same story with moving it to an info box.
```suggestion
// It may be counterintuitive that in tests you need to use `loadCKEditorCloud` instead of `useCKEditorCloud`.
// The reason for this is that `useCKEditorCloud` is a React hook and can only be used in React components,
// while tests are typically written as functions in testing suites. Therefore, in tests, you should use
// the `loadCKEditorCloud` function to load CKEditor 5 from the CKEditor Cloud and obtain the necessary
// CKEditor 5 objects. This allows you to properly test your CKEditor 5 integration without any issues.
```